Common/DTO/BrokerRequests/InternalCreateUserRequest.cs

```diff
@@ -1,8 +1,11 @@
-﻿namespace DTO.BrokerRequests
+﻿using System;
+
+namespace DTO.BrokerRequests
 {
     public class InternalCreateUserRequest
     {
         public User User { get; set; }
         public UserCredential Credential { get; set; }
+        public Guid Id { get; set; }

```

Comment:
Where do we user id in create user?

Reply:
Why do we have id here? Hello, it is already exist User and UserCredential

Reply:
Oh yes, I will fix it